internal/proxy/task_index.go

```diff
@@ -141,28 +141,27 @@ func (cit *createIndexTask) parseIndexParams() error {
 			indexParamsMap[kv.Key] = kv.Value
 		}
 	}
-	if !isVecIndex {
-		specifyIndexType, exist := indexParamsMap[common.IndexTypeKey]
-		if cit.fieldSchema.DataType == schemapb.DataType_VarChar {
-			if !exist {
-				indexParamsMap[common.IndexTypeKey] = DefaultStringIndexType
-			}
 
-			if exist && !validateStringIndexType(specifyIndexType) {
-				return merr.WrapErrParameterInvalid(DefaultStringIndexType, specifyIndexType, "index type not match")
-			}
-		} else if typeutil.IsArithmetic(cit.fieldSchema.DataType) {
-			if !exist {
-				indexParamsMap[common.IndexTypeKey] = DefaultArithmeticIndexType
-			}
+	indexType := indexParamsMap[common.IndexTypeKey]
```

Comment:
should check  common.IndexTypeKey exist ?

Reply:
If it not exist, the value is "".